components/src/core/components/TinyMce/TinyMce.vue

```diff
@@ -154,6 +156,43 @@ export default defineComponent({
           };
           reader.onerror = (error: any) => {};
         };
+
+        setTinymceBackGroundImage.value = (file: File) => {
```

Comment:
Is there any reason you made `setTinymceBackGroundImage` a ref? I don't see any place its used in template or script in a [reactive](https://vuejs.org/guide/essentials/reactivity-fundamentals) manner. I think it could be a simple const

Reply:
function calling place is inside the tinymce setup and function is outside of that scope